compat/vcbuild/vcpkg_install.bat

```diff
@@ -53,9 +62,18 @@ REM ================================================================
 	IF ERRORLEVEL 1 ( EXIT /B 1 )
 
 	echo Successfully installed %cwd%vcpkg\vcpkg.exe
+	GOTO :install_libraries
```

Comment:
Maybe we should not squeeze the `check_for_updates` "routine" between the part of the script where `vcpkg.exe` is built and `install_libraries`? That this extra `goto` is now required all of a sudden is a pretty clear indicator that this is the wrong location for `check_for_updates`. It would be much better to move it to the end of the script, as it is a pretty independent piece of new code.